packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/http-server
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+import {RestServer, RestComponent} from '@loopback/rest';
+import {Application, ApplicationConfig} from '@loopback/core';
+import * as assert from 'assert';
+
+describe('HttpServer', () => {
+  it('starts server', async () => {
+    const server = await givenAServer();
+    await server.start();
+    assert(server.listening, 'Server not started');
```

Comment:
This does not really verify that a server is started. You need to make an HTTP request and verify that it was handled as expected. Please use supertest as the client, perhaps by using `@loopback/testlab` - see https://github.com/strongloop/loopback-next/blob/master/packages/testlab/src/client.ts#L11-L13.

We also don't use `assert` from Node.js core because it has poor ergonomic and unhelpful error messages. Please use `expect` provided by testlab, see https://github.com/strongloop/loopback-next/tree/master/packages/testlab#basic-use and http://shouldjs.github.io